docs/docs/maintaining-a-plugin.md

```diff
@@ -1,10 +1,50 @@
 ---
 title: Maintaining a Plugin
-issue: https://github.com/gatsbyjs/gatsby/issues/21067
 ---
 
-This is a stub article meant to be filled with tips on how to maintain a Gatsby plugin once you've published it as an npm package.
+The Gatsby community thrives on the power of plugins, you often find that there is a plugin for almost everything ranging from source plugins, to transformer plugins, to plugins that are plugins for other plugin. There is a guide on the [naming conventions for plugins](/docs/naming-a-plugin/), check it out to learn more.
 
-Topics to be covered:
+> Wondering how to [create a plugin](/docs/creating-plugins) Look no further. Start contributing.
 
-- yarn workspaces can solve yarn link inconsistencies
+## What is a plugin?
+
+Gatsby plugins are Node.js packages that implement Gatsby APIs. [Learn more about plugins](/docs/plugins/)in this part of the Gatsby documentation.
+
+The bulk of the work is during the development phase. However, there’s still a need to take a look at the dependencies and security features from time to time. Keeping Plugins up to date becomes really tasking and it is important to be careful when updating dependencies seeing as this could potentially break your apps.
+
+However, the effects of not updating and maintaining can bring a huge security issue to your apps that you don't want . Let's look at some tips to apply when maintaining plugins:
```

Comment:
```suggestion
However, the consequences of not updating and maintaining a plugin can introduce security issues to your apps. The following are some recommendations for maintaining plugins:
```
It's a bit round about but "let's" = let us which is using "we" instead of "you". So I recommend rephrasing it a bit to adhere to style guide tone :)